examples/ssd/README.md

```diff
@@ -0,0 +1,25 @@
+# Examples of Single Shot Multibox Detector [1]
+
+## Performance
+PASCAL VOC2007 Test
+
+| Model | Original | ChainerCV |
+|:-:|:-:|:-:|
+| SSD300 | 77.5 % [2] | 77.5 % |
+| SSD512 | 79.5 % [2] | 79.6 % |
```

Comment:
I prefer more explicit notation.
I found that `%` is used, but it is used as something like `% mAP`.